docs/source/release/v4.3.0/mantidworkbench.rst

```diff
@@ -39,8 +39,9 @@ Improvements
 
 Bugfixes
 ########
-- Fixed an issue with Workspace History where unrolling consecutive workflow algorithms would result in only one of the algorithms being unrolled.
 
+- Fixed an issue with Workspace History where unrolling consecutive workflow algorithms would result in only one of the algorithms being unrolled.
+- Fixed a couple of errors in the python scripts generated from plots for newer versions of Matplotlib.
```

Comment:
I checked and it doesn't look like this was an issue in the last release so release notes shouldn't be needed.

Reply:
It is worth including as this has been reported by instrument scientists from the nightly